src/__tests__/ceramic_integration.test.ts

```diff
@@ -387,6 +388,16 @@ describe('Ceramic Integration Test', () => {
         const doc2 = await TileDocument.load(ceramic2, doc1.id)
         await doc2.update(updatedContent, null, { anchor: false })
 
+        // Make sure that the ceramic CAS has received the newest version
+        const casDocRef = await casCeramic1.loadStream(doc1.id)
+        await firstValueFrom(
+          casDocRef.pipe(
+            filter((state) => {
+              return state.next?.content.foo === updatedContent.foo
```

Comment:
I'm worried about if this condition is never hit the whole test will hang.  Can we add a 30 second timeout to this?  Might want to make a helper function that can be used in both tests and given a condition to check for in the state as well as a timeout

Reply:
yup will do